ViAn/GUI/mainwindow.cpp

```diff
@@ -932,13 +1050,53 @@ void MainWindow::remove_bookmarks_of_project(MyQTreeWidgetItem* project_item) {
  * removes the bookmarks in the bookmark_view
  * @param video_item to remove bookmarks from
  */
-void MainWindow::remove_bookmark_of_video(QTreeVideoItem* video_item) {
+void MainWindow::remove_bookmarks_of_video(QTreeVideoItem* video_item) {
     for(auto it = video_item->bookmarks.begin(); it != video_item->bookmarks.end(); it++){
         bookmark_view->remove_bookmark(video_item->id, *it);
     }
 }
 
 /**
+ * @brief MainWindow::remove_analysis_of_project
```

Comment:
Can we make an analysis of a project?
Do we really need both this and the next function?